src/Console/Command/ProcessCommand.php

```diff
@@ -140,8 +150,9 @@ private function ensureSomeRectorsAreRegistered(): void
      */
     private function processFiles(array $fileInfos): void
     {
-        $this->symfonyStyle->title('Processing files');
-        $this->symfonyStyle->progressStart(count($fileInfos));
+        $totalFiles = count($fileInfos);
+        $this->symfonyStyle->title(sprintf('Processing %s Files', $totalFiles));
```

Comment:
Here it should be probably `%d`, due to numeric value

Reply:
And `Process x **f**iles`, since it's informative message and not a headline in post-way

Reply:
Always forget that `sprintf` works like `C` printf :sweat_smile: